src/sentry/search/events/fields.py

```diff
@@ -2788,9 +2803,6 @@ def __init__(
                 ),
                 # TODO: implement these
                 SnQLFunction("histogram", snql_aggregate=self._resolve_unimplemented_function),
-                SnQLFunction("percentage", snql_aggregate=self._resolve_unimplemented_function),
-                SnQLFunction("t_test", snql_aggregate=self._resolve_unimplemented_function),
-                SnQLFunction("minus", snql_aggregate=self._resolve_unimplemented_function),
```

Comment:
Ended up implementing these specifically for the trends endpoint instead of including it on the general QueryBuilder